docs/src/visualization.md

```diff
@@ -0,0 +1,133 @@
+prior = combine_distributions([prior_u1, prior_u2])
+
+N_ensemble = 30
+initial_ensemble = construct_initial_ensemble(prior, N_ensemble)
+ekp = EnsembleKalmanProcess(initial_ensemble, y, Γ, Inversion(), verbose = true)
+
+N_iterations = 10
+for i = 1:N_iterations
+    params_i = get_ϕ_final(prior, ekp)
+
+    G_matrix = hcat(
+        [G(params_i[:, i]) for i = 1:N_ensemble]..., # Parallelize here!
+    )
+
+    update_ensemble!(ekp, G_matrix)
+end
+```
+
+## Plot priors
+
+The function `plot_priors` will plot marginal histograms for all dimensions of
+the parameter distribution.
+
+```@example plots
+using CairoMakie # load a Makie backend
+import EnsembleKalmanProcesses.Visualize as viz
+
+fig_priors = CairoMakie.Figure(size = (500, 400))
+viz.plot_priors(fig_priors[1, 1], prior)
+
+fig_priors
+```
+
+## Plot errors
+
+The functions `plot_error_over_iters` and `plot_error_over_time` and the
+mutating versions can be used to plot the errors over the number of iterations
+or time respectively. All keyword arguments supported by
+[`Makie.lines`](https://docs.makie.org/dev/reference/plots/lines) are supported
+by these functions too.
+
+Different errors can be plotted by the `error_metric` keyword argument. See
+[`compute_error!`](@ref) for a list of different error metrics.
+
+```@example plots
+# Assume that ekp is the EnsembleKalmanProcess object
+using CairoMakie # load a Makie backend
+import EnsembleKalmanProcesses.Visualize as viz
+
+fig_errors = CairoMakie.Figure(size = (400 * 2, 400 * 1))
+viz.plot_error_over_iters(fig_errors[1, 1], ekp, color = :tomato)
+# Error plotting functions support plotting different errors through the
+# error_metric keyword argument
+ax1 = CairoMakie.Axis(fig_errors[1, 2], title = "Average RMSE over iterations", yscale = Makie.pseudolog10)
+EnsembleKalmanProcesses.Visualize.plot_error_over_time!(
+    ax1,
+    ekp,
+    linestyle = :dashdotdot,
+    error_metric = "avg_rmse",
+)
+fig_errors
+```
+
+## Plot constrained parameters
+
+The functions `plot_ϕ_over_iters` and `plot_ϕ_over_time` and the mutating
+versions can be used to plot a scatter plot of the constrained parameters over
+the number of iterations or time respectively. All keyword arguments supported
+by [`Makie.Scatter`](https://docs.makie.org/dev/reference/plots/scatter) are
+supported by these functions too.
+
+Furthermore, there are also `plot_ϕ_mean_over_iters` and `plot_ϕ_mean_over_time`
+and the mutating versions can be used to plot the mean constrained parameters
+over the number of iterations or time respectively. All keyword arguments
+supported by [`Makie.Scatter`](https://docs.makie.org/dev/reference/plots/lines)
+are supported by these functions too. Furthermore, there is the keyword argument
+`plot_std = true` which can be used to also plot the standard deviation as well.
+To support plotting both mean and standard deviation, there are also the keyword
+arguments `line_kwargs` and `bend_kwargs` for adjusting how the mean and
```

Comment:
Could you explain what `bend_kwargs` are, with a link to Makie docs if applicable?